sdk/cpp/core/src/entity.cpp

```diff
@@ -40,25 +40,19 @@ Entity::~Entity()
 {
 }
 
-unique_ptr<Entity> Entity::clone_ptr()
+shared_ptr<Entity> Entity::clone_ptr()
 {
     return nullptr;
 }
 
-void Entity::set_child(const std::string & yang_name, Entity* entity)
```

Comment:
Yes, it has been removed after the `children` is changed to a public member. Previously, `children` is `protected` in `Entity`, but somehow I cannot make it compile with a pybind11 `def_readwrite`. So I changed it to `def_property` and added these `set_child`, and `get_child` in `entity.cpp`.